redaxo/src/core/lib/util/file.php

```diff
@@ -30,8 +30,28 @@ public static function get($file, $default = null)
      */
     public static function getConfig($file, $default = [])
     {
-        $content = self::get($file);
-        return $content === null ? $default : rex_string::yamlDecode($content);
+        if (is_readable($file)) {
+            $mtime = filemtime($file);
+            $filePathMd5 = md5($file);
+
+            $cacheFile = rex_path::coreCache($filePathMd5. '.yamlcache');
+            $cacheTimestampFile = rex_path::coreCache($filePathMd5. '.yamlcache.timestamp');
```

Comment:
Warum nicht direkt mit `filemtime($cacheFile)` arbeiten?
Bei unseren separaten Yaml-Caches machen wir das so. Siehst du da Probleme?
Beispiel: https://github.com/redaxo/redaxo/blob/master/redaxo/src/core/boot.php#L87

Reply:
Weil es so einfach ist das ich nicht drann gedacht hab 😎

Reply:
Ah nee. Es ist so wie jetzt, weil in dem .timestamp file drehen steht welche mtime das originalfile hatte als der cache generiert wurde.

Reply:
Verstehe noch nicht, was der Unterschied zum verlinkten Fall oben ist?
Du könntest doch auch prüfen, ob mtime cachefile neuer als mtime origdatei, und wenn ja, cache ausliefern, ansonsten  nicht.

Reply:
Hmm vllt haben wir das problem im core ja auch... :-)

UserA: package.yml wird gelesen
UserB: package.yml wird verändert
UserA: cache wird geschrieben

-> cache ist neuer als package.yml, entspricht aber nicht deren inhalt

Reply:
Hmm, so rein theoretisch hast du vermutlich recht.
Praktisch wird es vermutlich eher nicht vorkommen, da es meist nur einen Admin gibt, oder die Admins wie bei uns zum Beispiel lokal arbeiten.

Aber habe natürlich nichts dagegen, wenn wir es dann besser machen, sodass das Problem nicht auftauchen kann.